lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -219,8 +219,13 @@ defmodule MembraneLiveWeb.EventChannel do
     {:noreply, socket}
   end
 
-  def handle_in("reaction", _data, socket) do
-    broadcast(socket, "animation", %{})
+  def handle_in("heart_reaction", _data, socket) do
+    broadcast(socket, "heart_animation", %{})
+    {:noreply, socket}
+  end
+
+  def handle_in("confetti_reaction", _data, socket) do
+    broadcast(socket, "confetti_animation", %{})
     {:noreply, socket}
   end
```

Comment:
What do you think about a change like this?
```suggestion
  def handle_in(animation_type <> "_reaction", _data, socket) when animation_type in ["confetti", "heart"] do
    broadcast(socket,animation_type  <>"_animation", %{})
    {:noreply, socket}
  end
```

Reply:
Looks great! I'll commit it right away

Reply:
Although the code looks good, it does not work. The `<>` operator needs to have an explicit byte_size of all the arguments excluding the last one. Fixed that by switching animation type with `"reaction"` (it's `reaction_heart` now, not `heart_reaction`).